packages/atlas/src/views/studio/YppDashboard/tabs/YppDashboardSettingsTab.tsx

```diff
@@ -29,31 +46,54 @@ const categoriesSelectItems: SelectItem[] =
 
 export const YppDashboardSettingsTab = () => {
   const mdMatch = useMediaMatch('md')
+  const navigate = useNavigate()
   const { displaySnackbar } = useSnackbar()
+  const { channelId, memberId, accountId } = useUser()
   const { currentChannel, refetchSyncedChannels } = useGetYppSyncedChannels()
+  const { joystream, proxyCallback } = useJoystream()
   const [openModal, closeModal] = useConfirmationModal()
 
+  const handleTransaction = useTransaction()
+  const { dataObjectStateBloatBondValue } = useBloatFeesAndPerMbFees()
+  const channelBucketsCount = useChannelsStorageBucketsCount(channelId)
+
   const [isSync, setIsSync] = useState(currentChannel?.shouldBeIngested)
-  const [category, setCategory] = useState<string | null | undefined>(currentChannel?.videoCategoryId)
+  const [categoryId, setCategoryId] = useState<string | null | undefined>(currentChannel?.videoCategoryId)
   const areSettingsChanged = currentChannel
-    ? currentChannel.videoCategoryId !== category || currentChannel?.shouldBeIngested !== isSync
+    ? currentChannel.videoCategoryId !== categoryId || currentChannel?.shouldBeIngested !== isSync
     : false
   useEffect(() => {
     if (currentChannel) {
-      setCategory(currentChannel.videoCategoryId)
+      setCategoryId(currentChannel.videoCategoryId)
       setIsSync(currentChannel.shouldBeIngested)
     }
   }, [currentChannel])
 
   const handleChangeSettings = useCallback(async () => {
-    if (!currentChannel) return
+    if (!accountId || !joystream) {
+      SentryLogger.error('No joystream instance', 'YppDashboardSettingsTab')
+      return
+    }
+
+    if (!currentChannel || isSync === undefined) {
+      SentryLogger.error("Couldn't fetch current channel", 'YppDashboardSettingsTab')
+      return
+    }
     try {
+      const message: IngestChannelMessage = {
+        shouldBeIngested: isSync,
+        timestamp: Date.now(),
+        ...(categoryId ? { videoCategoryId: categoryId } : {}),
+      }
+
+      const signature = await joystream.signMessage({
+        data: JSON.stringify(message),
+        type: 'payload',
+      })
+
       const data = await axios.put(
-        `${atlasConfig.features.ypp.youtubeSyncApiUrl}/channels/${currentChannel.joystreamChannelId}`,
-        {
-          category,
-          isSync,
-        }
+        `${atlasConfig.features.ypp.youtubeSyncApiUrl}/channels/${currentChannel.joystreamChannelId}/ingest`,
+        { message, signature }
       )
 
       if (data.status === 200) {
```

Comment:
Looks like refetch is not working as intended. After changing the category footer is still visible.

![image](https://user-images.githubusercontent.com/92513933/211501909-0551a2c0-ecf8-4fd7-8c08-e6eecae79384.png)

Reply:
Same applies for `isSync`